modules/openapi-generator/src/main/resources/typescript-axios/baseApi.mustache

```diff
@@ -2,10 +2,11 @@
 /* eslint-disable */
 {{>licenseInfo}}
 
-import { Configuration } from "./configuration";
+import type { Configuration } from './configuration';
+import type { AxiosPromise, AxiosInstance, AxiosRequestConfig } from 'axios';
 // Some imports not used depending on template conditions
 // @ts-ignore
-import globalAxios, { AxiosPromise, AxiosInstance, AxiosRequestConfig } from 'axios';
```

Comment:
This causes a bug because now it no longer falls under the ts-ignore. Please make sure to add ts-ignore above line 6.  We're now stuck on a previous version

Reply:
(I created a PR to fix the issue: #14011)